course_content/cartopy_course/solutions/cartopy_final_exercise.ipynb

```diff
@@ -65,19 +59,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
-   "metadata": {
-    "collapsed": false
-   },
+   "execution_count": 5,
+   "metadata": {},
    "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/h02/esadek/.conda/envs/testenv/lib/python3.9/site-packages/scipy/__init__.py:146: UserWarning: A NumPy version >=1.16.5 and <1.23.0 is required for this version of SciPy (detected version 1.23.1\n",
+      "  warnings.warn(f\"A NumPy version >={np_minversion} and <{np_maxversion}\"\n"
+     ]
+    },
```

Comment:
We don't want this block in something that others will be looking at for answers in future. Could you remove it? Probably easiest by editing the text directly (rather than editing within a notebook).